tests/integration/security/main_test.go

```diff
@@ -60,5 +60,6 @@ func setupConfig(cfg *istio.Config) {
 components:
   egressGateways:
   - enabled: true
+    name: istio-egressgateway
```

Comment:
Is this required, or just "best practice"?

Reply:
it's required in principle and will also be in practice when we add validation for it.